### PR TITLE
Add fluent-cap-ctl command to handle add/drop/get/clear capability operations

### PR DESCRIPTION
**Pull request by cosmo0920**
Please review #3155 first.

**Which issue(s) this PR fixes**: 
None. Follows up #3155

**What this PR does / why we need it**: 
Linux capability operation should be handled in Fluentd management command.
This should be unified interface to add/drop/get/clear operations for Ruby executables which are used to execute Fluentd supervisor and worker(s).

**Docs Changes**:
Needed.  ~~I'll send a PR to describe this command.~~
https://github.com/fluent/fluentd-docs-gitbook/pull/249

**Release Note**: 
Same as title.

### CODE REVIEW

**Comment by repeatedly** [comment]
`fluent-cap-ctl --clear-cap` seems to be redundant. Need `-cap` suffix in the option?

**Comment by cosmo0920** [comment]
> `fluent-cap-ctl --clear-cap` seems to be redundant. Need `-cap` suffix in the option?

Hmm..., right. I'll remove `-cap` suffix from `--clear-cap` option.

**Comment by repeatedly** [comment]
How about other options? keep `-cap` is better? I'll leave it to you.

File: lib/fluent/command/cap_ctl.rb
Comment (repeatedly) [inline lib/fluent/command/cap_ctl.rb:35]: `from` instead of `into`?

**Review by repeatedly — APPROVED** [review]

**Comment by cosmo0920** [comment]
I'll merge this PR after CIs are finishing.

**Pull request merged**
